pages/index.html

```diff
@@ -1,12 +1,66 @@
+
+
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Theresa Davis</title>
+        <link rel="stylesheet" href="../styles/style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    </head>
+
+    <body>
+
+        <header class = "header-main">
+            <nav class = "nav-wrapper">
+              <div class = "title">
+                <h1>T</h1>
+                <h1>H</h1>
+                <h1>E</h1>
+                <h1>R</h1>
+                <h1>E</h1>
+                <h1>S</h1>
+                <h1>A</h1>
+                <h1>Davis</h1>
+                
+              </div>
+              <ul>
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./portfolio.html">Portfolio></a></li>
```

Comment:
```suggestion
                <li><a href="./portfolio.html">Portfolio</a></li>
```
there is an extra closing brace here